luup files/L_RGBController1.lua

```diff
@@ -1037,6 +1037,49 @@ function doColorTransition(lul_device)
 	end
 end
 
+-- MySensor RGB(W)
+RGBDeviceTypes["MYS-RGBW"] = {
+	getParameters = function (lul_device)
+		return {
+			name = "MySensors RGBW",
+			settings = {
+				{ variable = "ArduinoId", name = "Arduino plugin Id", type = "string" },
+				{ variable = "RadioId",   name = "RGB Node altid",    type = "string" }
+			}
+		}
+	end,
+
+	getColorChannelNames = function (lul_device)
+		return {"red", "green", "blue", "warmWhite", "coolWhite"}
```

Comment:
I've created a new branch with your code and some modifications.
Should (I hope) work from OOB with Arduino plugin.
As I haven't this plugin, I can not really test.

Please report if this try is successful :)